### PR TITLE
gh-96455: update example in exception_handling_notes.txt to the 3.11RC bytecode

### LINKED ISSUE #96455: exception_handling_notes is out of date
Objects/exception_handling_notes.txt has an example with out of date 3.11 bytecode output - it still has 3 pops to remove an exception from the stack, and the function call opcodes are out of date.

### PR DESCRIPTION
**Pull request by iritkatriel**
Fixes #96455.

* Issue: gh-96455

### CODE REVIEW

**Review by MatthieuDartiailh — COMMENTED** [review]
On line 91 of the patched version there is still a mention of pushing the exception as three values. If a single POP_TOP is necessary I assume the exception is pushed as a single value, no ?

**Comment by iritkatriel** [comment]
> On line 91 of the patched version there is still a mention of pushing the exception as three values. If a single POP_TOP is necessary I assume the exception is pushed as a single value, no ?

Thanks - I've fixed that now.

File: Objects/exception_handling_notes.txt [hunk 1]
```diff
@@ -43,33 +43,35 @@ SETUP_FINALLY and POP_BLOCK.
...
+        >>   48 COPY                     3
+             50 POP_EXCEPT
+             52 RERAISE                  1
 ExceptionTable:
-  2 to 8 -> 14 [0]
-  14 to 20 -> 28 [3] lasti
+  4 to 32 -> 38 [0]
+  38 to 40 -> 48 [1] lasti
 
-(Note this code is from an early 3.11 alpha, the NOP may well have be removed before release).
```
Comment (markshannon) [inline Objects/exception_handling_notes.txt:68]: Keep the note that this is from 3.11, so we don't have to update it all the time.
Maybe `(Note this code is from 3.11, later versions may have slightly different bytecode)`?

**Review by markshannon — COMMENTED** [review]
One suggestion, otherwise LGTM.

**Review by markshannon — APPROVED** [review]

**Pull request merged**